extensions/amp-story/1.0/amp-story.js

```diff
@@ -2634,9 +2634,57 @@ export class AmpStory extends AMP.BaseElement {
       this.next_();
     } else if (data['previous']) {
       this.previous_();
+    } else if (data['delta']) {
```

Comment:
Are we the only ones that use this method? Then we could do previous and next using delta -1 and +1, so the current approach (with previous and next) is obsolete